src/test/java/uk/gov/hmcts/ccd/domain/service/common/SecurityClassificationServiceTest.java

```diff
@@ -228,6 +229,58 @@ void shouldReturnCaseWhenUserHigherClassification() {
 
```

Comment:
There is a method in getCaseAccessCategory() in  SecurityClassificationServiceImpl.java  which contemplate  CaseAccessCategory null or "" .    Could we add an small test for it . Only because Sonar 

inal Map<String, JsonNode> data = JacksonUtils.convertValue(MAPPER.readTree(
                "{  \"CaseAccessCategory\":  NOTHING or not present.
                    "    }\n"
            ));